docs/01-usage/configuration.md

```diff
@@ -156,7 +156,7 @@ Example (`.releaserc` file content):
     ["@semantic-release/exec", {
       "verifyConditionsCmd": "./verify.sh"
     }],
-    "@semantic-release/git",
+    "@semantic-release/github",
     "@semantic-release/gitlab",
```

Comment:
The example is using gitlab, not github. The `@semantic-release/git` is not the same as `@semantic-release/github`